lib/axon.ex

```diff
@@ -2783,11 +2892,10 @@ defmodule Axon do
       param_byte_size = num_params * div(bitsize, 8)
 
       op_inspect =
-        case {op, opts[:layer_op]} do
-          {op, _} when is_atom(op) -> Atom.to_string(op)
-          {_, nil} -> "custom"
-          {_, layer_op} when is_atom(layer_op) -> Atom.to_string(layer_op)
-          {_, layer_op} when is_binary(layer_op) -> layer_op
+        if op_name do
+          Atom.to_string(op_name)
+        else
+          "custom"
         end
```

Comment:
If we have a default, we don't need to worry about checking `nil` consistently :)